sklearn/cluster/tests/test_affinity_propagation.py

```diff
@@ -244,14 +276,15 @@ def test_affinity_propagation_convergence_warning_dense_sparse(centers):
     rng = np.random.RandomState(42)
     X = rng.rand(40, 10)
     y = (4 * rng.rand(40)).astype(int)
-    ap = AffinityPropagation(random_state=46)
+    ap = AffinityPropagation(random_state=42)
```

Comment:
For my education: why sometimes use `0` and sometimes `42` as fixed seed? The OCD part of my brain was expecting it to always be one of the two :D

Reply:
In general we indeed ten to hardcode 0 or 42 when the intended meaning is "arbitrariness" (and when we don't need to explicitly check for seed independence with `global_random_seed`). However we are not very consistent so I think it's fine in leaving other seeds like 46.